packages/aws-cdk-lib/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -38,8 +38,16 @@ export interface QueueProcessingServiceBaseProps {
 
   /**
    * The image used to start a container.
+   *
+   * For QueueProcessingFargateService:
+   * Either 'image' or 'taskDefinition' must be specified, but not both.
+   *
+   * For QueueProcessingEc2Service:
+   * 'image' is required and must be specified.
+   *
+   * @default - none
```

Comment:
```suggestion
   * For `QueueProcessingFargateService`, either `image` or `taskDefinition` must be specified, but not both.
   * For QueueProcessingEc2Service, `image` is required.
   *
   * @default - the image of the task definition is used for Fargate, required otherwise
```
Cleanup.